Cargo.toml

```diff
@@ -13,8 +13,8 @@ readme = "README.md"
 
 [dependencies]
 keyberon-macros = { version = "0.1.0", path = "./keyberon-macros" }
-either = { version = "1.6", default-features = false }
+either = { version = "1.9.0", default-features = false }
 embedded-hal = { version = "0.2", features = ["unproven"] }
-usb-device = "0.2"
-heapless = "0.7"
-arraydeque = { version = "0.4.5", default-features = false }
+usb-device = "0.3.1"
```

Comment:
I'm a bit woried about this one, as it depends a lot of the hal updates. Also, there is already a dedicated PR for this.